libs/utils/test.py

```diff
@@ -42,20 +42,31 @@ def _init(cls, conf, *args, **kwargs):
             cls.logger.setLevel(kwargs['loglevel'])
             kwargs.pop('loglevel')
 
-        cls.conf = conf
-
         cls._runExperiments()
 
     @classmethod
+    def _getTestConf(cls):
+        if not hasattr(cls, "test_conf"):
+            raise NotImplementedError("Please add `test_conf` attribute")
+        return cls.test_conf
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        if not hasattr(cls, "experiments_conf"):
+            raise NotImplementedError("Please add `experiments_conf` attribute")
+        return cls.experiments_conf
+
+    @classmethod
     def _runExperiments(cls):
         """
         Default experiments execution engine
         """
 
         cls.logger.info("%14s - Setup tests execution engine...", "LisaTest")
-        test_env = TestEnv()
+        test_env = TestEnv(test_conf=cls._getTestConf())
 
-        cls.executor = Executor(test_env, experiments_conf=cls.conf);
+        exp_conf = cls._getExperimentsConf(test_env)
+        cls.executor = Executor(test_env, experiments_conf=exp_conf)
```

Comment:
Why not s/exp_conf/experiments_conf/ and pass it straight to the Executor?

```python
experiments_conf = cls._getExperimentsConf(test_env)
cls.executor = Executor(test_env, experiments_conf)
```

Thus avoiding to mix named and positional parameters in Executor ctor which also gets only these two params.